sites/singtel.com/singtel.com.config.js

```diff
@@ -0,0 +1,43 @@
+const dayjs = require('dayjs')
+const utc = require('dayjs/plugin/utc')
+
+dayjs.extend(utc)
+
+module.exports = {
+  site: 'singtel.com',
+  days: 2,
+  request: {
+    cache: {
+      ttl: 60 * 60 * 1000 // 1 hour
+    }
+  },
+  url({ date }) {
+    return `https://www.singtel.com/etc/singtel/public/tv/epg-parsed-data/${date.format('DDMMYYYY')}.json`
+  },
+  parser({ content, channel }) {
+    let programs = []
+    const items = parseItems(content, channel)
+    items.forEach(item => {
+      const start = dayjs(item.startDateTime)
```

Comment:
```suggestion
      const start = dayjs.utc(item.startDateTime)
```

I guess that's the problem.

Reply:
I did pass when I tested it, but I guess that's because I'm UTC anyhow.

Fixed